<launch>
	<arg name="veh" default="$(env VEHICLE_NAME)"/>
	<group ns="$(arg veh)">
		<remap from="circle_drive/car_cmd" to="lane_controller_node/car_cmd"/>
		<remap from="circle_drive/rotation" to="planning_node/rotation" />
		<remap from="circle_drive/end_trajectory" to="planning_node/end_trajectory" />
		<remap from="circle_drive/end_trajectory_sub" to="trajectory/end_trajectory_sub" />
		<node name="circle_drive" pkg="circle_drive" type="circle_drive.py" output="screen" required="true"/>

	</group>
</launch>
